Amplify/Categories/Auth/Models/AuthUserState.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public enum AuthUserState {
+
+    case signedIn
+
```

Comment:
No need for a `case guest` or something like that?

Reply:
Added.